skimage/filters/thresholding.py

```diff
@@ -1177,3 +1178,63 @@ def threshold_multiotsu(image, classes=3, nbins=256):
     thresh = bin_centers[thresh_idx]
 
     return thresh
+
+
+def threshold_phansalkar(image, window_size=15, k=0.25, r=None, p=2.0, q=10.0):
+    """Applies Phansalkar local threshold to an array. Phansalkar is a
+    modification of Sauvola technique to deal with low contrast images.
```

Comment:
There is an empty line missing between the short and the long description here and between the section further below as well. You can find [a few examples here](https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html) and in the [official reference](https://numpydoc.readthedocs.io/en/latest/) for the formatting we use throughout the library.

Reply:
I hope i was able to resolve that issue. I added some empty lines for readabilty as sown in the examples.